Kickstarter-iOS/Views/Controllers/ProjectPageViewController.swift

```diff
@@ -0,0 +1,415 @@
+import KsApi
+import Library
+import Prelude
+import UIKit
+
+public final class ProjectPageViewController: UIViewController, MessageBannerViewControllerPresenting {
+  // MARK: Properties
+
+  private enum NavigationButtonSizes: CGFloat {
+    case spacing = 15.0
+  }
+
+  private let viewModel: ProjectPageViewModelType = ProjectPageViewModel()
+  private let shareViewModel: ShareViewModelType = ShareViewModel()
+  private let watchProjectViewModel: WatchProjectViewModelType = WatchProjectViewModel()
+  /**
+   FIXME: This `contentController` can be renamed `contentViewController` and has to be embedded in a `PagingViewController` in https://kickstarter.atlassian.net/browse/NTV-195
+   Maybe check `BackerDashboardViewController`'s pageViewController for in-app examples on how to do this.
+   */
+  private var contentController: ProjectPamphletContentViewController?
+
+  // MARK: User session state properties
+
+  private var sessionEndedObserver: Any?
```

Comment:
Not sure what this is used for yet but I'm assuming this was carried over from the original implementation?

Reply:
Yea good call, can probably remove these and use `NotificationCenter.default` just like `didBackProject`